resolwe_bio/processes/workflows/wgbs.yml

```diff
@@ -48,6 +65,61 @@
           description: |
             Sets the number of reads to mapping in each loop. Larger number results in program
             taking more memory. This is especially evident in paired-end mapping.
+        - name: spikein_name
+          label: Chromosome name of unmethylated control sequence
+          type: basic:string
+          default: '' 
+          description: |
+            Specifies the name of unmethylated control sequence which is output as a separate alignment file.
+            It is recomended to remove duplicates to reduce any bias introduced by incomplete conversion
+            on PCR duplicate reads.
+        - name: filter_spikein
+          label: Remove control/spike-in sequences.
+          type: basic:boolean
+          default: false
+          description: |
+            Remove unmethylated control reads in the final alignment based on the provided name. It is recomended to remove
+            any reads that are not naturally occuring in the sample (e.g. lambda virus spike-in).
+    - name: bsrate
+      label: Bisulfite conversion rate
```

Comment:
If `spikein_name` is not provided, the whole `bsrate` group could be hidden with 
```
    - name: bsrate
      label: Bisulfite conversion rate
      hidden: >
        !alignment.spikein_name
      group:
            ...
```
Am I correct in thinking that if no `spikein_name` is given, this whole section of inputs is not relevant?